packages/moonstone/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/Scroller` to restore focus properly when newly target is container in voice control
```

Comment:
This fix seems to affect lists as well. If yes, please replace `moonstone/Scroller` with `moonstone/Scroller`, `moonstone/VirtualList.VirtualGridList`, and `moonstone/VirtualList.VirtualList`.

Reply:
I did fix the CHANGELOG.md. Thanks~